README.md

```diff
@@ -142,16 +168,13 @@ This project is licensed under the MIT License.
 
 ### Video Tutorial
 
-- [Accepting Payment With Paystack In React Native](https://www.youtube.com/watch?v=M-V4Q9zk9DE&t=19s) by [just1and0](https://twitter.com/just1and0) 
+- [Accepting Payment With Paystack In React Native](https://www.youtube.com/watch?v=M-V4Q9zk9DE&t=19s) by [just1and0](https://twitter.com/just1and0)
 
-## Sponsorship
-- Star the project on Github
-- [Buy me a coffee](https://buymeacoffee.com/6pL0Q8YkW)
+### Don't forget to star, like and share :)
 
-## Thanks to Our Superheroes ✨
-A huge shoutout to our amazing contributors! Your efforts make this project better every day. Check out the ([emoji key](https://allcontributors.org/docs/en/emoji-key)) for what each contribution means:
```

Comment:
Most of the changes you made to this isn't necessary, you are changing titles and descriptions which has nothing to do with improvements. Your change should only affect the improvement to code you are trying to make.


Also I think this PR might be boasted, it carries to many changes from code to docs. Can you make this more streamlined?